artifacts/camara-cloudevents/event-subscription-template.feature

```diff
@@ -0,0 +1,366 @@
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_27_expired_access_token_for_create_subscription
+  Scenario: Expired access token for create subscription
+    Given a valid <xxx> subscription request body and header "Authorization" is expired
+    When the request "create<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_28_invalid_access_token_for_create_subscription
+  Scenario: Invalid access token for create subscription
+    Given a valid <xxx> subscription request body 
+    And header "Authorization" set to an invalid access token
+    When the request "create<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscription_33_invalid_url
+  Scenario: Subscription creation with invalid url
+    Given a valid <xxx> subscription request body 
+    When the request "create<xxx>Subscription" is sent
+    And the request property "$.protocol" is set to "HTTP"
+    And the request property "$.sink" is set to "azerty"
+    Then the response property "$.status" is 400
+    And the response property "$.code" is "INVALID_SINK"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription retrieval scenarios #####################################
+
+  @<xxx>_subscriptions_29_no_authorization_header_for_get_subscription
+  Scenario: No Authorization header for get subscription
+    Given header "Authorization" is not present
+    And path parameter "subscriptionId" is set to the identifier of an existing <xxx> subscription
+    When the request "retrieve<xxx>Subscription" is sent 
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_30_expired_access_token_for_get_subscription
+  Scenario: Expired access token for get subscription
+    Given the header "Authorization" is set to expired token
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+    
+  @<xxx>_subscriptions_31_invalid_access_token_for_get_subscription
+  Scenario: Invalid access token for get subscription
+    Given the header "Authorization" set to an invalid access token
+    When the request "retrieve<xxx>Subscription" is sent

```

Comment:
Missing step: And path parameter "subscriptionId" is set to the identifier of an existing subscription